.vscode/launch.json

```diff
@@ -64,87 +61,87 @@
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Sync All",
             "args": ["--sync", "all", "--debugpath", "${input:debugPath}", "--debugbranch", "${input:debugBranch}"],
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Push Binaries",
             "args": ["--push", "Binaries.zip", "--debugpath", "${input:debugPath}"],
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Pull Binaries",
```

Comment:
Just writing that one here to keep it on sight, while running pull binaries configuration

```
10-May-20 14:14:50 [ERROR]  argument of type 'NoneType' is not iterable
Traceback (most recent call last):
  File "c:\Users\dyanikoglu\Desktop\PBSync\pbpy\pbhub.py", line 58, in pull_binaries
    if f"Downloading {binary_package_name}" in output:
TypeError: argument of type 'NoneType' is not iterable
10-May-20 14:14:50 [ERROR]  Exception thrown while trying do pull binaries for 0.63.19
10-May-20 14:14:50 [ERROR]  Failed to pull binaries for 0.63.19
```

Reply:
Should be fixed, test again.